sources/lib/util/options.ml

```diff
@@ -132,12 +133,18 @@ let vno_backward = ref false
 let vno_sat_learning = ref false
 let vinstantiate_after_backjump = ref false
 let vdisable_weaks = ref false
-let vinput_format = ref "why"
 let vanswers_with_loc = ref true
 
-type output = ONative | OSmtlib | OSZS
+(** Alt-Ergo infers automatically the input format according to the
+    file extension and set the corresponding parser. *)
+let vinfer_input_format = ref true
```

Comment:
Two things here:
- No need for doc comments in .ml files (i.e. use single `*` after the opening parenthesis).
- the comment as it is is not helpful from an implementation perspective because it doesn't answer the following question: what does `vinfer_input_format` does ? rather than a generic comment like that, something better would be:
`vinfer_input_format controls whether input format should be inferred by alt-ergo. If true, the vinput_format is basically ignored until format inferrence is done at which point it is set <I suppose ?>; if false, what happens ?(fill in what should actually happen / the semantic)`. Additionally, if possible some reference about where it is intended to be set and used would be interesting but not obligatory. And the same should be done for `vinput_format`.

Reply:
Basically, references and generally things which are done via side effects and whose changes should be synchronised should have documentation about what each state means, particularly corner cases where for instance, for some reasons (such as if specified by the user options on the command line), inference is activated **and** a specific input format is set: will it be ignored, will it supersede inference, etc...

Reply:
There's a button just above the comment box that allows you to make a suggestion that can immediately be committed (available with ctrl+g). Try to use it, it makes the review easier and faster to read ;-)

Reply:
There's a button just above the comment box available with ctrl+g as well which allows you to suggest a commit immediately. It makes the job easier for the person requesting your review and it's easier to read for other reviewers. Try to use it when you have a direct change in mind like this one where you provided what you think the comment should be.

Reply:
I'll keep that in mind for when it is applicable, but the problem is I don't know precisely what the message should be ! And that's exactly the problem: I don't know the semantics of these two references and how they are intended to interact with each other, and the current message does not answer these questions.